src/AudacityApp.cpp

```diff
@@ -373,6 +377,35 @@ void PopulatePreferences()
    gPrefs->Flush();
 }
 
+#if defined(USE_BREAKPAD)
+void InitBreakpad()
+{
+    wxFileName databasePath;
+    databasePath.SetPath(wxStandardPaths::Get().GetUserLocalDataDir());
+    databasePath.AppendDir("crashreports");
+    databasePath.Mkdir(wxS_DIR_DEFAULT, wxPATH_MKDIR_FULL);
+    
+    if(databasePath.DirExists())
+    {
+#if defined(WIN32)
+        wxFileName senderPath(wxFileName(wxStandardPaths::Get().GetExecutablePath()).GetPath(), "crashreporter.exe");
+#else
+        wxFileName senderPath(wxFileName(wxStandardPaths::Get().GetExecutablePath()).GetPath(), "crashreporter");
+#endif
```

Comment:
Comments elsewhere about the string literal.

Reply:
I think the literal should be mentioned in only one place, as a compiler command-line macro definition supplied by CMake, then just use the macro.

And perhaps not use it at all under src/ but make it private to crashreports.